src/classes/json_data.py

```diff
@@ -153,13 +157,28 @@ def read_from_file(self, file_path, path_mode="ignore"):
                 contents = f.read()
             if not contents:
                 raise RuntimeError("Couldn't load {} file, no data.".format(self.data_type))
-            
-            if os.path.splitext(file_path) and os.path.splitext(file_path)[1] == '.osp':
-                # Repair lost quotes on json keys
-                contents = re.sub('(\n\s*)(\w*):', r'\1"\2":', contents)
+
+            # Scan for and correct possible OpenShot 2.6.0 corruption
+            if self.damage_re_windows_drives.search(contents) and self.version_re_260.search(contents):
```

Comment:
Fixing just the projects from when the bug was present! Looks great.

Earlier you mentioned `verison <= 2.6.0` (but that might mean the file's already been parsed if we know the version.)

There may be a handful of users windows users who had the daily build in the last month, and had files on different drives. (A swiss cheese bug to be sure) But 
`r'"openshot-qt".*"2.(5..|6.0)')`

Reply:
I merged the windows drive letter fix right before creating the release branch. So I think we are probably good... minimal people this might impact. If so, they can manually update the "2.5.1-dev3" in the *.osp file to "2.6.0". Pretty easy work around, if anyone finds themselves in this situation